charmcraft/templates/init/README.md.j2

```diff
@@ -9,17 +9,15 @@ TODO: Describe your charm in a few paragraphs of Markdown
 TODO: Provide high-level usage, such as required config or relations
 
 
-## Developing
+## Relations
 
-Create and activate a virtualenv with the development requirements:
+TODO: Provide any relations which are provides or required by your charm
 
-    virtualenv -p python3 venv
-    source venv/bin/activate
-    pip install -r requirements-dev.txt
+## OCI Images
 
-## Testing
+TODO: Include a link to the default image your charm uses
 
-The Python operator framework includes a very nice harness for testing
-operator behaviour without full deployment. Just `run_tests`:
+## Contributing
 
-    ./run_tests
+Please see the [Juju SDK docs](https://juju.is/docs/sdk) for guidelines 
```

Comment:
“ following best practices.”

Reply:
are the underscores in requires and provides mandatory or do they fall under best practices?

Reply:
"Following best practice guidelines" is in the next line here.